src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5927,4 +5927,7 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ExternEventInitializer" xml:space="preserve">
     <value>'{0}': extern event cannot have initializer</value>
   </data>
+  <data name="ERR_ImplicitIndexerWithName" xml:space="preserve">
+    <value>Implicit {0} indexer cannot have a named argument.</value>
```

Comment:
>indexer cannot have a named argument. [](start = 24, length = 37)

I think the message is somewhat confusing. An indexer can have named parameters, in fact all parameters have names in C#. An **invocation** of the indexer cannot use named arguments.